packages/PlanLimitsUI/src/pages/Limits/sidebar.tsx

```diff
@@ -143,7 +144,12 @@ export default function Sidebar({
               mouseState.surfaceWaterMgmtSubUnitLimit
             }
             activeZonesIds={mouseState.groundWaterZones}
-            groundWaterZoneGeoJson={queries[6].data}
+            groundWaterZoneGeoJson={
+              queries[6].data as FeatureCollection<
+                Geometry,
+                GroundwaterZoneBoundariesProperties
+              >
+            }
```

Comment:
It's not great that we have to do this here rather than `/api/index.ts`, but it would require a bit more verbosity than I was happy with. I think we can keep chipping away at this, and in the next step move away from the array positioned queries to hash, and push the type declaration back into the API at that point.

Reply:
Yip fair enough, still a net win given how succinct you've made the code in `/api /index.ts`